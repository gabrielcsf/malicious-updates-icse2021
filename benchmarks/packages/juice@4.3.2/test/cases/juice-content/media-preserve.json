{
    "url": "./",
    "removeStyleTags": true,
    "preserveMediaQueries": true,
    "insertPreservedExtraCss": true,
    "preserveFontFaces": false,
    "extraCss": "@media all { body { color: blue; } }"
}
