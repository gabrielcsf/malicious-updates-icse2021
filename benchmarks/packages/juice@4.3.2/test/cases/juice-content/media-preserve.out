<html>
<head>
    <style>
@media screen {
  a {
    color: blue !important;
    background: red;
  }

  @font-face {
    font-family: 'Arial2';
  }
}
</style>
<style>
@media all {
  body {
    color: blue;
  }
}
</style></head>
<body>
    <p style="color: blue;">hi</p>
</body>
</html>
